src/btouch.cs

```diff
@@ -682,3 +625,40 @@ static void FixRelativeReferences (string sdk_path, List<string> references)
 		}
 	}
 }
+
+class SearchPathsAssemblyResolver : MetadataAssemblyResolver
+{
+	private readonly string[] libraryPaths;
+	private readonly string[] references;
```

Comment:
This is really minor, but code style - we don't use `private` when it's the default (Mono Coding Guidelines):

```suggestion
	readonly string[] libraryPaths;
	readonly string[] references;
```